aim/web/ui/src/components/CustomTable/Table.tsx

```diff
@@ -220,80 +220,93 @@ function Table(props) {
             'Table--grouped': groups,
           })}
         >
+          {!groups && props.multiSelect && Array.isArray(props.data) && (
+            <ErrorBoundary key={'selection'}>
+              <div
+                className={classNames({
+                  Table__pane: true,
```

Comment:
Write as a default class name instead of a permanent `true` parameter. `classNames` function is receiving the `default class names` string as the first parameter.